### PR TITLE
Unit Labels

### LINKED ISSUE #157: pokemon description units
**Is your feature request related to a problem? Please describe.**
Pokemon should display number value along with units, as per pokeapi documentation. 
https://github.com/PokeAPI/pokeapi.co/pull/27/files

image

### PR DESCRIPTION
**Pull request by Gjhnsn**
## Changes
_List Changes Introduced by this PR_
1. Updated measurement unit labels.

## Purpose
Provide a unit of measurement for description stats

Closes #157

### CODE REVIEW

File: src/components/PokemonDetails.js [hunk 1]
```diff
@@ -4,9 +4,16 @@ const PokemonDetails = ({ genera, flavorText, height, weight, abilities }) => {
...
+        {!flavorText ? (
+          <h3
+            style={{
+              margin: "50px auto",
+            }}>
+            Loading...
+          </h3>
+        ) : (
+          flavorText
+        )}
```
Comment (dannysanchez559) [inline src/components/PokemonDetails.js:16]: much more readable

**Review by connielion — APPROVED** [review]
Noice.

**Review by dannysanchez559 — APPROVED** [review]
gotta love those hectograms and decimeters

**Review by krferrell — APPROVED** [review]
If you want to convert, I thinki these function may be helpful: https://github.com/gonzalespaulb/Pokemon-App/pull/44/files

**Review by JorgeLVilla — APPROVED** [review]
Nice job Garret!

**Pull request merged**